### DISCUSSION TITLE
Run website in local

### DISCUSSION THREAD

**Discussion by winterwolf45** [MrGreensWorkshop/MrGreen-JekyllTheme#110] [Problems] [answered]
Hi I trying run this project in locally
But when I write command: **bundle exec jekyll serve --watch --host 0.0.0.0 --safe**
I have this error:
vagrant@fedora MrGreen-JekyllTheme]$ bundle exec jekyll serve --watch --host 0.0.0.0 --safe
Ignoring commonmarker-0.23.5 because its extensions are not built. Try: gem pristine commonmarker --version 0.23.5
Ignoring unf_ext-0.0.8.2 because its extensions are not built. Try: gem pristine unf_ext --version 0.0.8.2
Could not find commonmarker-0.23.5, unf_ext-0.0.8.2 in any of the sources
Run `bundle install` to install missing gems.

Thank you for sharing this template  website .

**Comment by MrGreensWorkshop** [comment, marked as answer]
HI, I didn't test in fedora,
It's actually self explanatory, why don't you start installing these. "commonmarker-0.23.5" and "unf_ext-0.0.8.2"
More than that, I'm afraid I can't help about jekyll installation problems.
